packages/checkbox/src/useCheckbox.ts

```diff
@@ -1,23 +1,183 @@
-import { ref, computed, inject } from 'vue'
+import {
+  ref,
+  computed,
+  inject,
+  getCurrentInstance,
+  onMounted,
+  watch,
+} from 'vue'
+import { toTypeString } from '@vue/shared'
+import { UPDATE_MODEL_EVENT } from '@element-plus/utils/constants'
+import { ICheckboxGroupInstance, ICheckboxProps, PartialReturnType } from './checkbox'
 
-export const useCheckbox = () => {
+export const useCheckboxGroup = () => {
   //todo: ELEMENT
   const ELEMENT = null
   const elForm = inject('elForm', {})
   const elFormItem = inject('elFormItem', {}) as any
-  const _checkboxGroup = inject('CheckboxGroup', {}) as any
-  const focus = ref(false)
-  const isGroup = computed(() => _checkboxGroup && _checkboxGroup.name === 'ElCheckboxGroup')
-  const _elFormItemSize = computed(() => {
+  const checkboxGroup = inject<ICheckboxGroupInstance>('CheckboxGroup', {})
+  const isGroup = computed(() => checkboxGroup && checkboxGroup.name === 'ElCheckboxGroup')
+  const elFormItemSize = computed(() => {
     return (elFormItem || {} as any).elFormItemSize
```

Comment:
We might need to specify the `ElForm` interface since I saw lots of code like this just casting it to any is not quite what we need right?
Can you put it into a new file for formItems? Maybe just for temporary, when form gets finished, we could just replace it with this

Reply:
Also, can you use the `?.` syntax to access `elFormItemSize`?